db_stress_tool/db_stress_test_base.cc

```diff
@@ -1945,10 +1945,13 @@ Status StressTest::PrepareOptionsForRestoredDB(Options* options) {
   options->listeners.clear();
   // Avoid dangling/shared file descriptors, for reliable destroy
   options->sst_file_manager = nullptr;
-  // GetColumnFamilyOptionsFromString does not create customized merge operator,
-  // and comparator.
+  // GetColumnFamilyOptionsFromString does not create customized merge operator.
   InitializeMergeOperator(*options);
   if (FLAGS_user_timestamp_size > 0) {
+    // Check OPTIONS string loading can bootstrap the correct user comparator
+    // from object registry.
+    assert(options->comparator);
+    assert(options->comparator == test::BytewiseComparatorWithU64TsWrapper());
     CheckAndSetOptionsForUserTimestamp(*options);
```

Comment:
We can remove this too?

Reply:
Do you mean to remove `CheckAndSetOptionsForUserTimestamp`? Yes, this can be removed, but I think it's no harm to go through the incompatible checks for the udt feature contained  in this function?

Reply:
Yeah, since it only checks stress test flags. But up to you.

Reply:
I see what you mean. There are only flag related checks in that function while flags should have been checked already in the same run. That makes sense, I removed this call.